certif/app/services/session.js

```diff
@@ -19,4 +29,35 @@ export default class CurrentSessionService extends SessionService {
   handleInvalidation() {
     super.handleInvalidation('/connexion');
   }
+
+  async loadCurrentUserAndSetLocale(localeFromQueryParam) {
+    this._localeFromQueryParam = localeFromQueryParam;
+
+    await this.currentUser.load();
+
+    if (this.currentDomain.getExtension() === FRANCE_TLD) {
+      this._setLocale(FRENCH_INTERNATIONAL_LOCALE);
+
+      if (!this.locale.hasLocaleCookie()) {
+        this.locale.setLocaleCookie(FRENCH_FRANCE_LOCALE);
+      }
+
+      return;
+    }
+
+    if (this._localeFromQueryParam) {
+      this._setLocale(this._localeFromQueryParam);
+      return;
+    }
+
+    const isUserLoaded = !!this.currentUser.certificationPointOfContact;
```

Comment:
Question : on a `await this.currentUser.load();` en tout début de fonction, je m'attends donc à avoir un currentUser.
Dans quel cas on aurait cette condition `!!this.currentUser.certificationPointOfContact;` ?

Reply:
La récupération du user est maintenant gérée à l'extérieur de la méthode handleLocale (anciennement nommée handleLocaleFromQueryParam). Merci.